classes/db/DbPDO.php

```diff
@@ -60,6 +60,7 @@ protected static function _getPDO($host, $user, $password, $dbname, $timeout = 5
         } else {
             $dsn .= 'host='.$host;
         }
+        $dsn .= ';charset=utf8';
```

Comment:
Yeah but this class isn't symfony-aware so I don't think we can do much about it :\